src/structures/GuildChannel.js

```diff
@@ -334,8 +334,22 @@ class GuildChannel extends Channel {
       });
     }
 
-    const permission_overwrites =
-      data.permissionOverwrites && data.permissionOverwrites.map(o => PermissionOverwrites.resolve(o, this.guild));
+    let permission_overwrites = false;
```

Comment:
This should probably default to undefined, not false (so if either branches fail, we don't send anything to Discord